### PR TITLE
formulaire 4 tuto + code complet

### PR DESCRIPTION
**Pull request by GtAntoine**

### CODE REVIEW

File: src/services/index.js [hunk 1]
```diff
@@ -1 +1,16 @@
-// déclarez ici les appels aux API de services métier
+import focusFetch from 'focus-application/fetch/fetch-proxy'
+
+export const loadUserFinance = async ({id}) => {
+    const response = await focusFetch({url: `http://localhost:9999/x/complex/${id}`, method: 'get'})
+    const data = await response;
+    return {...data, __Focus__updateRequestStatus: data.__Focus__updateRequestStatus};
```
Comment (Ephrame) [inline src/services/index.js]: you can return data diectly

File: src/views/user/custom-finance-user-form.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    connectToFieldHelpers()
+)(SmartUser);
+
+const ConnectedFinanceForm = compose(
+    connectToMetadata(['financialMove', 'finance']),
+    connectToForm(formConfigFinance),
+    connectToFieldHelpers(),
+    connectToState(selectData('customData'))
+)(SmartFinance);
+
```
Comment (Ephrame) [inline src/views/user/custom-finance-user-form.js]: The ScrollSpyContainer is not necessary, it was just a test. Can you remove it ? No ? What do you think ?

File: src/views/user/custom-finance-user-form.js [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    return(
+        <Panel title='User' {...otherProps}>
+            {fieldFor('uuid')}
+            {fieldFor('firstName')}
+        </Panel>
+    )
+}
+
+const Finance = ({fieldFor, listFor, ...otherProps}) => (
+    <Panel title='Finance' {...otherProps}>
```
Comment (Ephrame) [inline src/views/user/custom-finance-user-form.js]: The entityPath is not necessary

**Review by Ephrame — COMMENTED** [review]
There is no tuto ?

File: src/views/user/custom-finance-user-form.js (hunk 2 above)
Comment (GtAntoine) [reply src/views/user/custom-finance-user-form.js]: I thought it was a good idea to let it as an exemple. I'd prefer to add a paragraph about it in the tutorial than remove it.

File: src/services/index.js [hunk 4]
```diff
@@ -1 +1,19 @@
-// déclarez ici les appels aux API de services métier
+import focusFetch from 'focus-application/fetch/fetch-proxy'
+
+export const loadUserFinance = async ({id}) => {
+    return focusFetch({url: `http://localhost:9999/x/complex/${id}`, method: 'GET'}).then((data) => {
+        return {
```
Comment (TomGallon) [inline src/services/index.js]: remove the THEN

File: src/views/user/custom-finance-user-form.js (hunk 2 above)
Comment (TomGallon) [reply src/views/user/custom-finance-user-form.js]: Ok, we keep it. Please add a sentence about it.

**Review by TomGallon — CHANGES_REQUESTED** [review]

**Pull request merged**